index.js

```diff
@@ -96,11 +105,9 @@ function init() {
       this.log.apply(this, args);
     },
 
-    err: function error() {
-      let args = Array.prototype.slice.call(arguments);
-      args.unshift('err');
-      this.log.apply(this, args);
-    },
+    err : error(),
+
+    error:  error(),
```

Comment:
Remove the parentheses on both `err` and `error`

Reply:
ook .. :)

Reply:
@mbrandau Its not possible ... test does not pass ... some semantic errors happen ... because errorFn actually returns a function ... so it has to be called with () ... :)

Reply:
@mbrandau Done ... :)